### PR TITLE
Airlines: New design and solved related issues

### PR DESCRIPTION
**Pull request by amoraleda**
The Airlines IA has been adapted to the new design, matching the Flights IA. Included in this PR, other previous bugs has been fixed.

- The bar is showing the scheduled time and the dot the actual flight progress
- Added a 5 minute margin to be considered "Delayed"

delayeddot

- Status information matching the size of the flight and date information

normal

- Show date beside the scheduled time if departure and arrival time is not in the same day
- Allowed relaxed queries for the FlightsIA #1838 

allowquerys

- Fixed other minor bugs

***
**IA Page**: https://duck.co/ia/view/airlines
Maintainer: @jdorweiler

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@amoraleda thanks for this, we'll get it deployed to beta for testing!

**Comment by jdorweiler** [comment]
Doesn't seem to work on beta.  I'll see if I can figure that out.

**Comment by jdorweiler** [comment]
Wrong IA page.  It's actually flights by route not airlines.  I'll update the IA page link in the first comment so it installs correctly.

**Comment by amoraleda** [comment]
@jdorweiler, thanks for the help. I think my description could've led to misunderstanding, but I'm pretty sure that it's the Airlines IA. I also added a couple of small changes to the Flight IA, as I thought it could be the best way.

**Comment by jdorweiler** [comment]
@amoraleda sorry, I get it now.  It's finally working on beta. 
selection_316

**Comment by moollaza** [comment]
@thm all the changes to Flights and Route are now live on beta: https://beta.duckduckgo.com/?q=klm+1662&ia=flights

**Comment by tagawa** [comment]
Has something happened? This is not showing for me on beta now.

**Comment by tagawa** [comment]
@amoraleda , thanks again for fixing the Airlines IA! You’ve been doing a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in travel.

When you're ready to move on, please:

- Create a new Instant Answer for searches you do related to travel

OR

- Test live travel Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Comment by jdorweiler** [comment]
Maybe @moollaza can take a look at the css changes? I'm not the best person to evaluate those.

File: share/spice/airlines/airlines.css [hunk 1]
```diff
@@ -1,61 +1,56 @@
...
+.zci--airlines .divider {
+    height: 1px;
+    border: 0;
+    margin: 1.2em 0;
 }
 
-.zci--airlines .tile__not {
-    color: #C14316;
+.zci--airlines .dot {
+	border-radius: 50%;
```
Comment (moollaza) [inline share/spice/airlines/airlines.css]: nit: looks like we've mixed tabs & spaces

**Comment by moollaza** [comment]
All this LGTM :+1:

**Comment by jdorweiler** [comment]
:+1:  thanks @amoraleda

**Pull request merged**